blog-cn/Group1:Deep analysis of PouchContainer technology.md

```diff
@@ -0,0 +1,94 @@
+# Deep analysis of PouchContainer's rich container technology
+
+PouchContainer is an efficient, open-source, lightweight Enterprise Rich Container Engine Technology developed by the Alibaba group. This technology presents the characteristics of strong isolation, high portability, and low resource consumption for a container. With the help of PouchContainer, corporations can rapidly deploy their business into containers and improve the utilization of physical resources in their data centers under hyperscale at the same time.
+
+PouchContainer originates from the thoughts of Alibaba's engineers to the problems they had in their business scenarios. These engineers have dedicated enormous time and efforts in delivering internet applications, which could be proved by the characteristics of the PouchContainer listed above. Thanks to the mass scale of Alibaba, this technology embraced its first opportunity to be fully tested on its ability to support business at Nov 11th after it comes to the world.  After declared as an open source project, this new technology is positioned to serve for the public and to help enterprises quickly realize the containerization of their business.
+
+![image1.png | center | 180x60](https://cdn.yuque.com/lark/0/2018/png/65333/1526059437042-3ae3f269-7795-44b2-a618-e6af68dc9f21.png "")
+
+When it first comes to container technology, Alibaba had an amazing amount of backlogs inside at that time. How to quickly containerize these backlogs is the key problem of this technology when it was first promoted inside the company. Nowadays, the open source container technology is becoming more and more popular. Many companies may still be worrying about how to deal with the same problem.  In the cloud native field, many of the advanced concepts promoted by the CNCF Foundation are based on business containerization. If enterprises fall behind this trend, they will be overwhelmed by the follow-up problems brought by container management, let alone enjoying the benefits of open source projects like Service Mesh.
```

Comment:
According to Oxford dictionary, backlog means "An accumulation of uncompleted work or matters needing to be dealt with". It is not accurate to describe the existing businesses Alibaba has. A better translation could be "legacy businesses"